### DISCUSSION TITLE
Is there a way to set the default polymorphic component without creating a new one?

### DISCUSSION THREAD

**Discussion by Californian** [mantinedev/mantine#5386] [Q&A] [answered]
I'm trying to use `next/image` as my polymorphic component for all the `Image`s in my app, but it looks like `defaultProps` doesn't accept a `component`. Did I miss something, or do I just have to wrap the component manually and import it?

**Comment by rtivital** [comment, marked as answer]
Type for `component` prop as well as other associated props that are inferred from it is handled with a generic interface, you cannot have component in default props and preserve type safety.
